atom/browser/api/atom_api_window.cc

```diff
@@ -26,10 +27,6 @@
 #include "native_mate/dictionary.h"
 #include "ui/gfx/geometry/rect.h"
 
-#if defined(TOOLKIT_VIEWS)
-#include "atom/browser/native_window_views.h"
-#endif
-
```

Comment:
I don't think this change is needed anymore, right? It's a leftover from when the code was static_casting window_ into a NativeWindowView